### ISSUE TITLE
Home Screen - Large Event Promo Cards

### ISSUE THREAD

**Issue by xavjones8** [NCSU-App-Development-Club/Marigold-Android#9]
# **Description**:

Design hardcoded large event promo cards. Create a new composable files for each Figma component in a new "home" package.

# **Accessibility Requirements**:
Text and content should be readable at maximum 200% screen zoom.
The accessibility label should announce the card as a button, and read off all the text in the card.

# **Links and Resources**:

Draft designs can be found here under the **************Xavier************** tab, but are not finalized

https://www.figma.com/file/raH7fqviVPnDFKILPBJ5iN/NCSU-Club-%26-Events-App?type=design&node-id=0-1&mode=design&t=vAJwOwAhwt5KSvGg-0

# **Attachments**:
![EventCardLarge](https://github.com/NCSU-App-Development-Club/Marigold-Android/assets/44850083/5fd29fda-2252-476b-920d-8e3c15c8b891)
